model/version_modification.go

```diff
@@ -27,7 +26,7 @@ func ModifyVersion(ctx context.Context, version Version, user user.DBUser, modif
 			return http.StatusInternalServerError, errors.Wrap(err, "restarting patch")
 		}
 	case evergreen.SetActiveAction:
-		if version.Requester == evergreen.MergeTestRequester && modifications.Active {
+		if version.Requester == evergreen.GithubMergeRequester && modifications.Active {
 			return http.StatusBadRequest, errors.New("commit queue merges cannot be manually scheduled")
```

Comment:
nit: might be better to rephrase to `"merge queue patches cannot be manually scheduled"` since the commit queue was deprecated.